core/src/scopypreferencespage.cpp

```diff
@@ -57,7 +57,7 @@ QWidget* ScopyPreferencesPage::createRestartWidget() {
 	lay->addSpacerItem(space);
 	lay->addWidget(btn,1);
 
-	w->connect(btn,&QPushButton::clicked, btn, [](){ApplicationRestarter::triggerRestart();});
+	w->connect(btn,&QPushButton::clicked, btn, [](){ApplicationRestarter::GetInstance()->triggerRestart();});
```

Comment:
By making the triggerRestart method static it makes it easier to call and improves readability.